src/components/common/settings/sidebar.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import {
+  Banknote,
+  Bell,
+  ChevronLeft,
+  Database,
+  Globe,
+  LucideIcon,
+  User,
+  UserPlus,
+} from "lucide-react";
+import Link from "next/link";
+import { usePathname, useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+
+import { cn } from "~/lib/utils";
+
+type SettingsSidebarProperties = {
```

Comment:
This type and the type anotation below isn't needed. TypeScript will infer the properties.